src/blog/2024/06/interacting-with-google-sheet-from-node-red.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Interacting with Google Sheet from Node-RED
+subtitle: Guide to learn how to write, read, update and delete data in Google sheet using Node-RED.
+description: Learn how to interact with Google Sheet from Node-RED to write, read , update and delete data.
+date: 2024-06-24
+authors: ["sumit-shinde"]
+image: 
+tags:
+   - posts
+   - node-red
+   - node red
+   - google sheet
+   - how to
+---
+
+Have you ever needed to integrate Google Sheets with your Node-RED application to track and manage data seamlessly? This guide will walk you through the process of integrating Google Sheets with Node-RED, enabling you to write, read, update, and delete data effortlessly.
+
+<!--more-->
+
+## What is the Google Sheet?
+
+[Google Sheets](https://www.google.com/sheets/about/) is a cloud-based spreadsheet application developed by Google. It allows users to create, edit, and collaborate on spreadsheets in real-time over the Internet. This makes it an ideal option for easily and securely collaborating on data that is not large in size. In businesses, Google Sheets is commonly used for tasks such as analyzing daily profits, tracking expenses, and managing collaborative projects. However, for products or services with large user bases, businesses often prefer using databases, which are recommended for efficiently managing and scaling data operations.
+
+## Prequisite
+
+Before proceeding, make sure you have installed the following node from the pallet manager.
+
+- [node-red-contrib-google-sheets](https://flows.nodered.org/node/node-red-contrib-google-sheets)
+
+## Interacting with Google Sheets with Node-RED
+
+When writing this blog, I watched some videos on the internet about how to integrate Google Sheets with our application with traditional programming. I noticed that almost all of the videos demonstrate complex steps and involve a lot of coding. Fortunately, as a Node-RED developer, you won't have to go through such lengthy processes.
+
+### Creating a Google service account 
```

Comment:
Not sure what these lines are for? :) It's a guide, lets focus on getting things done

```suggestion
```

Reply:
should we also remove the section's title ?

Reply:
I think so, but up to you.